docs/section11/Rule11-8.md

```diff
@@ -0,0 +1,65 @@
+# Service_Water_Heating - Rule 11-8
+**Schema Version:** 0.0.37  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-8
+
+**Rule Description:** "One system per building area type shall be modeled in the baseline."
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, a + b
+
+**Evaluation Context:** B-RMD each SWH BAT  
+**Data Lookup:**   
+**Function Call:**  
+- **get_SWH_equipment_associated_with_each_SWH_bat**  
+- **get_building_segment_SWH_bat**  
+- **get_SWH_bats_and_SWH_use**  
+
+
+## Applicability Checks:
+- Each SWH bat is applicable if there are SWH loads in the BAT in the proposed model
+
+- call the function get_SWH_bats_and_SWH_use for the proposed: `swh_bats_and_uses_p = get_SWH_bats_and_SWH_use(P_RMD)`
+- look through each of the SWH bats in the building: `for swh_bat in swh_bats_and_uses_p:`
+    - look at each service water heating use in the BAT: `for swh_use_id in swh_bats_and_uses_p[swh_bat]:`
+      - get the swh use from the swh_use_id: `swh_use = get_component_by_id(P_RMD, swh_use_id)`
+      - check if there are SWH loads in this use, continue to rule logic.  In this case, we are not worried about the use_units, we just want to know that there is a use that is greater than 0: `if swh_use.use > 0: CONTINUE TO RULE LOGIC`
+    - otherwise, rule is not applicable for this swh_bat: `else: NOT_APPLICABLE`
+
+        ## Rule Logic: 
+    - use get_SWH_equipment_associated_with_each_SWH_bat to get the SWH use types and SWH use in the building: `swh_bats_and_equip_dict = get_SWH_equipment_associated_with_each_SWH_bat(B_RMD)`
+    - get a dictionary of all SWH equipment connected to the current SWH bat in the baseline model: `swh_bats_and_equip_dict_this_use = swh_bats_and_equip_dict[swh_bat]`
+    - get the quantity of service water heating distribution systems serving this building area type: `num_swh_systems = len(swh_bats_and_equip_dict_this_use["SWHDistribution"])`
+    - get the quantity of service water heating equipment serving this building area type: `num_swh_equipment_this_use = len(swh_bats_and_equip_dict_this_use["SWHHeatingEq"])`
+    - create a boolean is_referenced_in_other_bats and set it to false: `is_referenced_in_other_bats = false`
+    
+    - if there is exactly one system, check that the system is not used in any of the other BATs: `if num_swh_systems == 1:`
```

Comment:
Just wrappign my head around this in writing. The reason that there is no need to also check SWHEquip is because equip can only reference one distribution system and if there is only one distribution system associated with the BAT and there is only one Equip object associated with the BAT then it guarantees that Equip will not be referenced by other BATs as long as the distribution system isn't referenced by other BATs.

Reply:
So, my understanding of this rule is that there can only be one distribution system per BAT.  There is no mention of only one waterheater - does this seem correct?

**ServiceWaterHeatingEquipment** can only be connected to one **ServiceWaterHeatingDistributionSystem**, so if the distribution system is only connected to one BAT, the equipment must also be only connected to one BAT, even if there are multiple.

Reply:
7/18 - discussed on call - conclusion that the requirement is for one "Service Water Heating System" though the definition is vague, and we should allow multiple Service Water Heating Equipment.